cmd/skaffold/app/cmd/build.go

```diff
@@ -53,14 +67,19 @@ func build(out io.Writer, filename string) error {
 		return errors.Wrap(err, "creating runner")
 	}
 
-	bRes, err := runner.Build(ctx, out, runner.Tagger, config.Build.Artifacts)
+	buildOut := out
+	if quietFlag {
```

Comment:
Maybe we can set this at the root level and quiet all commands by setting out = ioutil.Discard